database/migrations/2016_12_06_124120_create_skill_user_table.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateSkillUserTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('skill_user', function(Blueprint $table) {
```

Comment:
Nope: http://stackoverflow.com/questions/34274998/any-advantages-of-adding-an-id-column-to-a-pivot-table-in-laravel